File: Data Structures & Algorithms/jump-game/submission-2.cpp
class Solution {
public:
bool func(vector<int> &nums,int i,int n){
    if(i==n) return true;
    else if(i>n) return false;
    if(i<n && nums[i]==0) return 0;
    for(int j=1;j<=nums[i];j++){
        if(func(nums,i+j,n)) return true;
    }
    return false;
}
    bool canJump(vector<int>& nums) {

        int n=nums.size();
        return func(nums,0,n-1);
        
    }
};

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-2.py
from heapq import heappush,heappop
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        n=len(nums)
        heap=[]
        for a in nums :
            if len(heap) < k :
                heappush(heap,a)
            else :
                if a > heap[0] :
                    heappop(heap)
                    heappush(heap,a)
        
        return heap[0]
        